trade-finance-manager-api/src/v1/api.js

```diff
@@ -328,6 +328,35 @@ const getDealCancellation = async (dealId) => {
   }
 };
 
+/**
+ * Deletes the deal cancellation object on a TFM AIN or MIN deal
```

Comment:
Do we need to mentino AIN/MIN deal here? If MIA is added for example this could become out of sync/misleading

Reply:
MIA deals will be abandoned on a separate endpoint - I think this is a business logic thing & I dont expect it to change